azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTest.java

```diff
@@ -225,14 +234,8 @@ public void exec1FailedKillAll() throws Exception {
     Assert.assertTrue(runner.isKilled());
 
     Assert.assertTrue(
-        "Expected flow " + Status.FAILED + " instead " + exFlow.getStatus(),
-        exFlow.getStatus() == Status.FAILED);
-
-    try {
-        Thread.sleep(500);
```

Comment:
Thanks for improving this slow tests.
Any suggestion on how to make the tests run faster so that we can run them on every commit?